app/controllers/api/v1/rooms_controller.rb

```diff
@@ -24,7 +24,7 @@ def index
         shared_rooms = SharedAccess.where(user_id: current_user.id).select(:room_id)
         rooms = Room.where(user_id: current_user.id)
                     .or(Room.where(id: shared_rooms))
-                    .order(online: :desc, last_session: :desc)
+                    .order(Room.arel_table[:online].desc, Room.arel_table[:last_session].desc.nulls_last)
```

Comment:
That seems to be the best solution,
However, we should use `Arel` only when it's required.
 The `online` sorting can use the regular hash syntax.
`.order(online: :desc).order(Room.arel_table[:last_session].desc.nulls_last)`

Reply:
Yeah agreed

Reply:
Didnt think about chaining the `.order` thank you!

Reply:
You're welcome,